src/main/java/org/semver4j/internal/range/RangeProcessorPipeline.java

```diff
@@ -1,23 +1,28 @@
 package org.semver4j.internal.range;
 
+import org.jetbrains.annotations.NotNull;
 import org.semver4j.internal.range.processor.Processor;
 
 public class RangeProcessorPipeline {
+    @NotNull
```

Comment:
That's not necessary in favor of construct object with not-null parameters.

Reply:
Reasons 1 & 5